ws-nextjs-app/pages/[service]/new_live/[id]/Post/index.tsx

```diff
@@ -56,15 +59,51 @@ const PostBreakingNewsLabel = ({
   ) : null;
 };
 
+const TimeStamp = ({ curated }: { curated: string }) => {
+  const {
+    timezone,
+    locale,
+    altCalendar,
+    service,
+    articleTimestampPrefix,
+    articleTimestampSuffix,
+    script,
+  } = useContext(ServiceContext);
+
+  const isRelative = isTenHoursAgo(new Date(curated).getTime());
+
+  return (
+    <TimeStampContainer
+      css={styles.timeStamp}
+      timestamp={curated}
+      dateTimeFormat="YYYY-MM-DD"
+      locale={locale}
+      timezone={timezone}
+      service={service}
+      prefix={articleTimestampPrefix}
+      suffix={articleTimestampSuffix}
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
Rather than using `@ts-ignore`, can we use `@ts-expect-error` and provide a reason (as per [Component Guidelines](https://latest--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?viewMode=story&id=coding-standards-migrating-legacy-components--page&args=#use-ts-expect-error-instead-of-ts-ignore-to-suppress-unfixable--expected-typescript-errors))